doc/man3/EVP_MAC.pod

```diff
@@ -114,7 +115,8 @@ I<ctx>.
 =head2 Computing functions
 
 EVP_MAC_init() sets up the underlying context with information given
-through diverse controls.
+via the I<key> and I<params> arguments.  The MAC I<key> has a length of
+I<keylen> and the parameters in I<params> are processed before setting the key.
```

Comment:
Say something about what happens if the  key is NULL.

Reply:
Okay, will add.